manifests/t/Trimble/SketchUpPro2022Test/22.0.354/Trimble.SketchUpPro2022Test.locale.en-US.yaml

```
@@ -0,0 +1,20 @@
+# Created using wingetcreate 1.0.4.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Trimble.SketchUpPro2022Test

```

```suggestion
PackageIdentifier: Trimble.SketchUp.2022.Pro
```
Because there are different channels of this software (Go, Pro, Studio) and they appear to be versioned by year, it would be best to take an approach similar to that of Visual Studio and separate the package identifiers